Frontend/src/tests/home.test.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import Home from '../pages/home/Home';
+import { NavLink } from 'react-router-dom';
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  NavLink: jest.fn(({ children, to }) => (
+    <a href={to} data-testid={children}>
+      {children}
+    </a>
+  )),
+}));
+
+test('renders nav buttons', () => {
+  render(<Home />);
+
+  const menuItems: string[] = ['Meals', 'Shopping', 'Chores'];
+
+  menuItems.forEach((menuItem: string) => {
+    const navButton = screen.getByTestId(menuItem);
+    expect(navButton).toBeInTheDocument();
+    expect(navButton).toHaveTextContent(menuItem);
+  });
+});
```

Comment:
I think this test isn't testing anything. Please check this by changing something in the component you're testing and seeing if the test fails or not. For example:
Remove `data-testid` from the component and then run the test again. If it still passes then the test is not adding any value. 

It looks to me that you are overriding completely the component and then testing the override code and not the original.

Reply:
@immanuelAjz,
Where does it look like she's overriding the component?